changelog.d/4872.misc

```diff
@@ -0,0 +1 @@
+Enabling new FTUE Auth onboarding base, includes the "I already have an account" button in the splash
```

Comment:
wasn't sure if this was a `.feature` or `.misc` as we're only switching the build config value, could even be a new separate type (if we're able to have custom types)

Reply:
OK for misc. I think we are limited by towncrier for possible extension but maybe not.